src/annotator/anchoring/range-util.js

```diff
@@ -0,0 +1,227 @@
+import $ from 'jquery';
+
+/**
+ * A simple XPath evaluator using jQuery which can evaluate queries of
+ */
+function simpleXPathJQuery(nodes, relativeRoot) {
```

Comment:
Since we're in the process of removing jQuery, I would be inclined to remove this code path and just keep the non-jQuery one. It might be easier to do this first before landing the conversion of CoffeeScript => JS.